package.json

```diff
@@ -135,7 +132,6 @@
     ]
   },
   "resolutions": {
-    "esbuild": "^0.18.20",
-    "**/gl": "^6.0.2"
+    "**/gl": "^8.0.2"
```

Comment:
Hmm, @sayomaki this resolution seems to be breaking the CI again, which is weird, because locally, I can run the commands just fine.

This is different from frontend where I am faced with the same error locally.